applications.json

```diff
@@ -7497,6 +7497,25 @@
                 "cpp",
                 "objective_c"
             ]
+        },
+        {
+            "short_description": "Quickly share pasteboard to Gist, and files via iCloud or Dropbox",
+            "categories": [
+                "Productivity"
+            ],
+            "repo_url": "https://github.com/msztech/AirPasteboardOpen",
+            "title": "Air Pasteboard",
+            "icon_url": "https://github.com/msztech/AirPasteboardOpen/blob/master/AirPasteBoard/Assets.xcassets/StatusBarButtonImage.imageset/bar32w.png?raw=true",
+            "screenshots": [
+               "https://github.com/msztech/AirPasteboardOpen/raw/master/demo.gif",
```

Comment:
Please change to https://raw.githubusercontent.com/mszopensource/AirPasteboardOpen/master/demo.gif